### DISCUSSION TITLE
全闪Nvme SSD部署的seaweedfs FIO压测的IOPS只有几百

### DISCUSSION THREAD

**Discussion by Icedroid** [seaweedfs/seaweedfs#5151] [Q&A]
seaweedfs 后端是3个节点，每个节点8块7.6T的全闪Nvme SSD盘，通过filer +redis 提供文件存储服务，使用weed mount 挂载后的性能为什么那么低? 复制策略是001，三个节点所有盘都对应一个weed volume server, 使用同一dc 和 rack 
![image](https://github.com/seaweedfs/seaweedfs/assets/674647/d5d6ad70-a5e1-4f12-98c7-8e918624bd45)

**Comment by NyaMisty** [comment]
因为对象存储就不是让你随机写的。。。建议你自己先看看对象存储的应用场景。。。
而且谁家测对象存储的randrw用4k测啊？？？？
